packages/@o3r/extractors/src/core/wrapper.ts

```diff
@@ -12,6 +17,43 @@ export const createBuilderWithMetricsIfInstalled: BuilderWrapper = (builderFn) =
   try {
     const { createBuilderWithMetrics } = await import('@o3r/telemetry');
     wrapper = createBuilderWithMetrics;
-  } catch {}
+  } catch (e: any) {
+    // Do not throw if `@o3r/telemetry is not installed
+    const angularJsonPath = path.posix.join(ctx.workspaceRoot, 'angular.json');
+    const angularJson = existsSync(angularJsonPath) ? JSON.parse(readFileSync(angularJsonPath, {encoding: 'utf8'}).toString()) : undefined;
+
+    if (angularJson.cli?.o3rMetrics !== false) {
+      ctx.logger.debug('`@o3r/telemetry` is not available.\nAsking to add the dependencies\n' + e.toString());
+
+      const question: Question = {
+        type: 'confirm',
+        name: 'isReplyPositive',
+        message: `
+          Would you like to share anonymous usage data about this project with the Otter Team at Amadeus ?
+          It will help us to improve our tools.
+          For more details and how to change this settings, see https://github.com/AmadeusITGroup/otter/blob/main/docs/telemetry/README.md.
+        `
+      };
+      const { isReplyPositive } = await prompt([question]);
+
+      if (isReplyPositive) {
+        const pmr = getPackageManagerRunner(angularJson);
+
+        try {
+          execSync(`${pmr} ng add @o3r/telemetry`);
+        } catch {}
+
+        const { createBuilderWithMetrics } = await import('@o3r/telemetry');
+        return createBuilderWithMetrics(builderFn)(opts, ctx);
+      } else {
+        ctx.logger.info('You can activate it at anytime by running `ng add @o3r/telemetry`.');
+
+        angularJson.cli ||= {};
+        angularJson.cli.o3rMetrics = false;
```

Comment:
Not valid according to Angular.json schema:
https://github.com/angular/angular-cli/blob/main/packages/angular/cli/lib/config/workspace-schema.json#L88